src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/PersonIdentityDetailed.java

```diff
@@ -16,11 +16,18 @@ public class PersonIdentityDetailed {
     @JsonProperty("address")
     private final List<Address> addresses;
 
+    @JsonProperty("drivingPermit")
+    private final List<DrivingPermit> drivingPermits;
+
     public PersonIdentityDetailed(
```

Comment:
Can't we have a new constructor here, and leave the existing one. So we have the one with driving permits list

Reply:
Corrected.